### ISSUE TITLE
Zigbee2MQTT fails to start after 1.25.0

### ISSUE THREAD

**Issue by Mega-Admin** [Koenkk/zigbee2mqtt#12067]
### What happened?

Everything worked fine then updated to 1.25 and Zigbee2MQTT failed to start.

Restored backup for 1.24 and it is now also broken.

Config didn't change when updating to 1.25. I tried a couple of the solutions others have posted here but none worked.

```
data_path: /config/zigbee2mqtt
external_converters: []
devices: devices.yaml
groups: groups.yaml
homeassistant: true
permit_join: false
mqtt:
  base_topic: zigbee2mqtt
  server: mqtt://core-mosquitto:1883
  user: mqtt-user
  password: _removed_
serial:
  port: >-
    /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2485224-if00
  adapter: deconz
advanced:
  log_level: warn
  pan_id: 6754
  channel: 21
  network_key:
    _removed_
  availability_blocklist: []
  availability_passlist: []
device_options: {}
blocklist: []
passlist: []
queue: {}
frontend:
  port: 8099
experimental: {}
availability: false
socat:
  enabled: false
  master: pty,raw,echo=0,link=/tmp/ttyZ2M,mode=777
  slave: tcp-listen:8485,keepalive,nodelay,reuseaddr,keepidle=1,keepintvl=1,keepcnt=5
  options: '-d -d'
  log: false
```

### What did you expect to happen?

Zigbee2MQTT to update

### How to reproduce it (minimal and precise)

Update to 1.25

### Zigbee2MQTT version

1.25

### Adapter firmware version

Unknown

### Adapter

Conbee II

### Debug log

Log after updating to 1.25:

```
[11:15:48] INFO: Handing over control to Zigbee2mqtt Core ...
> zigbee2mqtt@1.25.0 start
> node index.js
Zigbee2MQTT:debug 2022-04-04 11:15:50: Loaded state from file /config/zigbee2mqtt/state.json
Zigbee2MQTT:info  2022-04-04 11:15:50: Logging to console and directory: '/config/zigbee2mqtt/log/2022-04-04.11-15-49' filename: log.txt
Zigbee2MQTT:debug 2022-04-04 11:15:50: Removing old log directory '/config/zigbee2mqtt/log/2022-04-04.11-13-36'
Zigbee2MQTT:info  2022-04-04 11:15:50: Starting Zigbee2MQTT version 1.25.0 (commit #unknown)
Zigbee2MQTT:info  2022-04-04 11:15:50: Starting zigbee-herdsman (0.14.20)
Zigbee2MQTT:debug 2022-04-04 11:15:50: Using zigbee-herdsman with settings: '{"adapter":{"concurrent":null,"delay":null,"disableLED":false},"backupPath":"/config/zigbee2mqtt/coordinator_backup.json","databaseBackupPath":"/config/zigbee2mqtt/database.db.backup","databasePath":"/config/zigbee2mqtt/database.db","network":{"channelList":[21],"extendedPanID":[221,221,221,221,221,221,221,221],"networkKey":"HIDDEN","panID":6754},"serialPort":{"adapter":"deconz","path":"/dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2485224-if00"}}'
CREATED DECONZ ADAPTER
Zigbee2MQTT:error 2022-04-04 11:16:00: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2022-04-04 11:16:00: Failed to start zigbee
Zigbee2MQTT:error 2022-04-04 11:16:00: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2022-04-04 11:16:00: Exiting...
Zigbee2MQTT:error 2022-04-04 11:16:00: undefined
```

Log after restoring backup to 1.24:

```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] socat.sh: executing... 
[11:17:35] INFO: Socat not enabled, marking service as down
[cont-init.d] socat.sh: exited 0.
[cont-init.d] zigbee2mqtt.sh: executing... 
[11:17:36] INFO: MQTT available, fetching server detail ...
[11:17:36] INFO: MQTT server settings not configured, trying to auto-discovering ...
[11:17:36] INFO: Configuring 'mqtt://core-mosquitto:1883' mqtt server
[11:17:36] INFO: Previous config file found, checking backup
[11:17:36] INFO: Creating backup config in '/config/zigbee2mqtt/.configuration.yaml.bk'
[11:17:36] INFO: Adjusting Zigbee2mqtt core yaml config with add-on quirks ...
[cont-init.d] zigbee2mqtt.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[11:17:37] INFO: Handing over control to Zigbee2mqtt Core ...
> zigbee2mqtt@1.24.0 start
> node index.js
CREATED DECONZ ADAPTER
Zigbee2MQTT:error 2022-04-04 11:17:49: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2022-04-04 11:17:49: Failed to start zigbee
Zigbee2MQTT:error 2022-04-04 11:17:49: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2022-04-04 11:17:49: Exiting...
Zigbee2MQTT:error 2022-04-04 11:17:49: undefined
```

**Comment by Bolukan** [comment]
I updated from 1.23.0 and the container did not start due to errors. I reverted back to 1.23.0 and will watch this topic how it is analysed and solved.

**Comment by dyarfaradj** [comment]
I got same exact issue running `Current version: 1.25.0-1`
`Version
core-2022.4.2 
Version
supervisor-2022.04.0`


`[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] socat.sh: executing... 
[20:48:28] INFO: Socat not enabled, marking service as down
[cont-init.d] socat.sh: exited 0.
[cont-init.d] zigbee2mqtt.sh: executing... 
[cont-init.d] zigbee2mqtt.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[20:48:39] INFO: Handing over control to Zigbee2mqtt Core ...
> zigbee2mqtt@1.25.0 start
> node index.js`

**Comment by Mega-Admin** [comment]
I've tried rebuilding from scratch and still no help. Anyone know the steps to clear all existing data to rebuild as if a fresh install?
I deleted all zigbee2mqtt files and uninstalled, rebuilt it and still getting the same errors.

I only have an zigbee network of about 10 devices and a few automations so this isn't a HUGE issue but I would prefer that I didn't have to. But a working zigbee setup is better than waiting on a response here.
